updatecli/updatecli.d/charts/jenkins-agents-infra.ci.jenkins.io.yaml

```
@@ -95,6 +95,7 @@ pullrequests:
   default:
     kind: github
     scmid: default
+    title: Bump agent templates version for infra.ci.jenkins.io (packer-image version {{ source packerImageVersion }})
```

```suggestion
    title: Bump agent templates for infra.ci.jenkins.io (packer-image v{{ source packerImageVersion }})
```
Reducing a little bit the length of the next PR title